prototype/Frontend/index.html

```diff
@@ -3,13 +3,51 @@
   <head>
     <meta charset="utf-8" />
     <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    
+    <!--
+      -----SEO META TAGS-----
+    -->
 
-    <title>The Weekly Challenge - Perl & Raku</title>
     <meta
-      content="The Weekly Challenge that includes Perl and Raku programming."
       name="description"
+      content="TheWeeklyChallenge: Unleash your coding potential with our engaging weekly challenges! Sharpen your skills, build projects, and join a thriving developer community."
+    />
+    <meta
+      name="keywords"
+      content="weekly coding challenges, programming practice, Perl and Raku, Dancer2 framework, online coding competitions"
+    />
+    <meta name="author" content="TheWeeklyChallenge.org Team" />
+    <meta name="robots" content="index, follow" />
+
+    <meta
+      property="og:title"
+      content="You don't need to be an expert, Come and hone Your Coding Skills Every Week"
+    />
+    <meta
+      property="og:description"
+      content="Take your coding skills to the next level with TheWeeklyChallenge. Participate in engaging weekly challenges, learn from peers, and build a strong developer portfolio."
     />
-    <meta content="The Weekly Challenge, Perl,Raku" name="keywords" />
+    <meta property="og:image" content="../assets/misc/logo.png" />
+    <meta property="og:url" content="https://theweeklychallenge.org" />
+    <meta name="twitter:card" content="../assets/misc/logo.png" />
```

Comment:
Don't use relative path instead use something like /public/.....

Reply:
Hello mentor!
I guess this will be done when we start converting the template ?

Because If I use the `/public` absolution path now, it will not be able to locate the logo file.
I hope I'm getting it wrong!

Reply:
Noted mentor ✅